package/doc/classes/index.default.md

```diff
@@ -0,0 +1,772 @@
+# Welcome to the VAPI docs
```

Comment:
The docs should be build automatically from code documentation (Docstrings). So they are not part of the git repo. Any need to be ignored in the `.gitignore`

Reply:
They are, its located as a script in the `package.json`. I'll write a github action to automatically make them and put them on a separate branch.

Reply:
Perfect